README.md

```diff
@@ -72,8 +84,12 @@ class MyDaemonizedCommand extends DaemonCommand
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $output->writeln("Iteration");
-
-        usleep(100000);
+        
+        // This method helps to give back the CPU to the react-loop (here for 1000 $µseconds).
+        // So you can wait between two iterations if your workers has nothing to do.
+        
+        // We do not recommand to use sleep() or usleep() anymore.
```

Comment:
> recommand
=> recommend


Is there somewhere else in the doc where we did recommend `sleep()` or `usleep()`?

because if it's true, we should also remove these recommendations

anyway, I'm not sure we should say that we recommended it before, it's in the `git history` forever now, they can look for older versions if they want to see our previous recommendations :)